### DISCUSSION TITLE
IGMP Proxy supported natively by UniFi?

### DISCUSSION THREAD

**Discussion by dmutsaers** [fabianishere/udm-iptv#231] [Q&A]
Hi,

I've read the following in the release notes of [UniFi Network Application 7.4.140](https://community.ui.com/releases/UniFi-Network-Application-7-4-140/7ca7417e-1893-482b-86c3-044c2382967f)

- Add support for IPTV through IGMP Proxy (for UXG/UniFi OS Consoles with gateway functions)

So UniFi supports IGMP Proxy natively now?

**Comment by tylerdola** [comment]
I am trying to get this to work. I have now updated to UDM-Pro 3.0.19 and Network 7.4.149... 
![image](https://user-images.githubusercontent.com/14016834/229319872-9fd2870d-2a67-4fac-a088-89c8a31bd641.png)

**Comment by tylerdola** [comment]
It works!! I disabled my udm-iptv and turned on this new feature.... works!!

**Reply by dmutsaers** [reply]
Good to hear! What about the following remark by @fabianishere ?

> Thanks for letting us know. I checked quickly, but it does not seem like it supports providers like KPN (where IPTV is on a separate VLAN) in its current form.

[https://github.com/fabianishere/udm-iptv/issues/223#issuecomment-1426761441](url)

**Reply by tylerdola** [reply]
> Good to hear! What about the following remark by @fabianishere ?
> 
> > Thanks for letting us know. I checked quickly, but it does not seem like it supports providers like KPN (where IPTV is on a separate VLAN) in its current form.
> 
> [https://github.com/fabianishere/udm-iptv/issues/223#issuecomment-1426761441](url)

I spoke too soon. I seem to be having a problem when I have multiple STB's on my VLAN 33. You change the channel on one, and the others freeze.

**Reply by perryvvliet** [reply]
Any update on this topic, does IGMP Proxy works with KPN IPTV?
I am running version Network 7.5.187
![image](https://github.com/fabianishere/udm-iptv/assets/131259435/8cf8550e-cff9-4a87-96bf-b1c7ffc863a7)

**Comment by dmutsaers** [comment]
I assume it still doesn't suppport IPTV on a seperate VLAN.

> Thanks for letting us know. I checked quickly, but it does not seem like it supports providers like KPN (where IPTV is on a separate VLAN) in its current form.

UNA 7.4.140 contains IGMP Proxy
https://github.com/fabianishere/udm-iptv/issues/223
